languages/ocaml/main.go

```diff
@@ -40,8 +41,16 @@ func Execute(config *utils.Config) {
 		panic(err)
 	}
 
+	args := []string{"ocaml"}
+	// Fix for getting the various libs added to prybar when running through nix
```

Comment:
:disappointed: I wonder why ocaml doesn't pick this up itself?

Reply:
I have no idea, I am using the latest one too :frowning_face: